### PR TITLE
Update create-a-custom-sensitive-information-type.md

### PR DESCRIPTION
**Pull request by pratjain523**
@chrfox Added a note to indicate which SITs can not be copied.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-a-custom-sensitive-information-type.md [hunk 1]
```diff
@@ -113,6 +113,18 @@ Use this procedure to create a new sensitive information type that is based on a
 13. Choose the **recommended confidence level** for this sensitive information type.
 14. Check your setting and choose **Submit**.
 
+> [!NOTE]
+> Following sensitive information types can not be copied.
+> - Canada Driver's License Number
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type.md]:
```suggestion
> The following sensitive information types can not be copied:
> - Canada driver's license number
```

File: microsoft-365/compliance/create-a-custom-sensitive-information-type.md [hunk 2]
```diff
@@ -113,6 +113,18 @@ Use this procedure to create a new sensitive information type that is based on a
...
+> - EU driver's license number
+> - EU national identification number
+> - EU passport number
+> - EU social security number or equivalent identification
+> - EU Tax identification number
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type.md]:
```suggestion
> - EU tax identification number
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pratjain523, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**